src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-troubleshooting.mdx

```diff
@@ -72,3 +72,29 @@ For Logs, you can use a structured log template to inject the `service.name`. He
 For more OpenTelemetry examples with New Relic, visit the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples) repository on GitHub.
 </Callout>
 
+## Troubleshooting the OpenTelemetry collector [#troubleshooting-collector]
+
+The best place for collector troubleshooting tips and monitoring practices is the up-to-date guidelines in the OpenTelemetry community.
```

Comment:
Do you have a link to the OTEL community? Like somewhere in GitHub?

Reply:
@barbnewrelic, the sections that followed have the links, so I added this line: See the links below for community troubleshooting documents.